.github/Dockerfile.ubuntu-22.04

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:22.04
+RUN echo 'APT::Install-Suggests "0";' >> /etc/apt/apt.conf.d/00-docker
+RUN echo 'APT::Install-Recommends "0";' >> /etc/apt/apt.conf.d/00-docker
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        git python3 python3-pip \
+        build-essential libssl-dev zlib1g-dev libcurl4-openssl-dev libexpat-dev gettext coreutils && \
+  pip install flake8
+ENV GIT_VERSIONS="v2.38.1 v2.25.5"
```

Comment:
I think we could also use the minimum required version here (2.20). We might want to get https://github.com/git-pile/git-pile/pull/99 merged first though. Your call.

Reply:
It's more because it's commonly-used distro - ubuntu 20.04

Alternative would be to simply use "container: ubuntu:22.04" / "container: xxx" and do the setup from there. Advantage with the approach taken here is that we don't have to repeat the entire tests just because of the change in git version. And also that we can eliminate some of our setup done at each execution, since now we created our own container.

Testing 2.20.x would be good, but I don't think it will work and we will have to raise it more.